app/app.py

```diff
@@ -102,7 +99,7 @@ def patch(cls, event: EventType) -> Tuple[int, Dict[str, str]]:
 
 
 def dispatch_request(event: EventType, context: ContextType) -> ResponseType:
-    request_path = event['requestContext']['path']
+    request_path = event['requestContext']['resourcePath']
```

Comment:
Good point 😸
But we've gotta modify unit tests as well, I'm gonna update them later.

Reply:
https://github.com/aws-samples/aws-rapid-prototyping-with-python/pull/1/commits/d3909e9ec7d3847efadc89b2d5696ae4c87f310a